src/rard/research/tests/models/test_bibliography.py

```diff
@@ -11,51 +10,54 @@ class TestBibliography(TestCase):
     def setUp(self):
         self.parent_object = TextObjectField.objects.create(content="foo")
 
-    def test_creation(self):
-        data = {
-            "authors": "Hartley, J. R.",
-            "title": "Fly Fishing",
-            "parent": self.parent_object,
-        }
-        item = BibliographyItem.objects.create(**data)
-        self.assertTrue(BibliographyItem.objects.filter(pk=item.pk).exists())
-
-    def test_parent_required(self):
+    def test_display(self):
+        # the __str__ function should show the name
         data = {
             "authors": "Hartley, J. R.",
+            "author_surnames": "Hartley",
             "title": "Fly Fishing",
+            "year": "1855",
         }
-        # cannot create a comment with no parent
-        with self.assertRaises(IntegrityError):
-            BibliographyItem.objects.create(**data)
+        b = BibliographyItem(**data)
+        self.assertEqual(str(b), "Hartley [1855]: Fly Fishing")
 
-    def test_delete_parent_deletes_item(self):
+    def test_creation(self):
         data = {
             "authors": "Hartley, J. R.",
             "title": "Fly Fishing",
-            "parent": self.parent_object,
         }
         item = BibliographyItem.objects.create(**data)
         self.assertTrue(BibliographyItem.objects.filter(pk=item.pk).exists())
-        self.parent_object.delete()
-        # NB this is only true when a GenericRelation exists on the
-        # parent model
-        self.assertFalse(BibliographyItem.objects.filter(pk=item.pk).exists())
 
     def test_related_query_name_for_bib_item_fields(self):
         data = {
             "authors": "Hartley, J. R.",
             "title": "Fly Fishing",
-            "parent": self.parent_object,
         }
         item = BibliographyItem.objects.create(**data)
```

Comment:
I think you could create this BibliographyItem in the setUp and call it something like `bi1` and then you wouldn't have to repeat the creation? It might help to look at the [AnonFrag tests](https://github.com/UCL/frrant/blob/70a6b1059367044369efc716cbd1a7d46c05565d/src/rard/research/tests/models/test_fragment.py#L91) or the [References tests](https://github.com/UCL/frrant/blob/70a6b1059367044369efc716cbd1a7d46c05565d/src/rard/research/tests/models/test_original_text.py#L209) (their methods are slightly different but the premise is the same)

Reply:
Sounds sensible, thanks for that - I need some more tests to fill the gaps, so I'll do it that way.